<launch>
    <arg name="dd_id_5" value="5"/>
    <node pkg="mpc_flock" name="dd_kinematics_$(arg dd_id_5)" type="dd_kinematics" output="screen" args="$(arg dd_id_5)">
        <param name="init_x_$(arg dd_id_5)" value="0.0"/>
        <param name="init_y_$(arg dd_id_5)" value="-4.0"/>
        <param name="init_theta_$(arg dd_id_5)" value="0.0"/>
    </node>

    <node pkg="mpc_flock" name="data_preparation_$(arg dd_id_5)" type="data_preparation" output="screen" args="$(arg dd_id_5)">
    </node>
    <node name="simple_distributed_nmpc_controller_$(arg dd_id_5)"
            pkg="simple_distributed_nmpc_controller"
            type="simple_distributed_nmpc_controller_node"
            respawn="true"
            clear_params="true"
            output="screen"
            args="$(arg dd_id_5)">
        <rosparam file="$(find simple_distributed_nmpc_controller)/config/open_params.yaml" />
    </node>
</launch>